client/src/components/SearchBox/SearchBox.component.jsx

```diff
@@ -69,7 +69,7 @@ const SearchBox = ({
     FullStory.event('Search', {
       timeToFirstSearch_int: timeToFirstSearch,
     })
-    googleAnalytics.setIsFirstSearch(true)
+    googleAnalytics.isFirstSearch = true
```

Comment:
given that `isFirstSearch` is state in the `GoogleAnalyticsProvider`, we should be using the `setIsFirstSearch` setter so that the change propagates correctly - see [useState docs](https://reactjs.org/docs/hooks-state.html)

Reply:
I changed it to a variable, because I'm not very convinced of the use of states when the value is only changed once and is only read after.